src/api/dao/dao-utils.js

```diff
@@ -18,9 +18,9 @@ daoUtils.isUserAdmin = (userInfo) => {
             let groupId = userInfo.groups[i];
             for (let groupIndex = 0; groupIndex < groups.groups.length; ++groupIndex) {
                 const group = groups.groups[groupIndex];
-                if (groupId != group.id) {
-                    continue;
```

Comment:
@darshan-clvt , why you removed existing code ? any reason ?

Reply:
you told me comment it out!..